checkers/rules/rules.go

```diff
@@ -771,3 +771,19 @@ func stringsCompare(m dsl.Matcher) {
 		`strings.Compare($s1, $s2) > 0`).
 		Suggest(`$s1 > $s2`)
 }
+
+//doc:summary   Finds unchecked errors in if statements
```

Comment:
All gocritic checkers doc comments should start with "Detects".

Reply:
+